CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
--
+- [added] Added `WebpushFcmOptions` to the `FirebaseMessaging` API, which 
+  provides `setLink()` method.
```

Comment:
Suggest a slight rephrasing as:

"...API, providing the `setLink()` method."

Alternatively, you could omit the second clause and just end with "...API."